libyul/optimiser/LoadResolver.cpp

```diff
@@ -146,10 +146,11 @@ void LoadResolver::tryEvaluateKeccak(
 		{
 			bytes contentAsBytes = toBigEndian(*memoryContent);
 			contentAsBytes.resize(static_cast<size_t>(*byteLength));
+			auto const val = u256(keccak256(contentAsBytes));
```

Comment:
Having two names in the same context that look like they are the same thing but actually aren't (`value`, `val`) is a bit of an anti-pattern. If you really want to extract this bit, please give it a meaningful name:

```suggestion
			u256 const contentHash(keccak256(contentAsBytes));
```